pkg/blockstorage/azure/azuredisk.go

```diff
@@ -375,12 +375,12 @@ func getRegionAndZoneID(ctx context.Context, s *adStorage, sourceRegion, volAz s
 		return region, "", nil
 	}
 
-	zones, err := zone.FromSourceRegionZone(ctx, s, sourceRegion, volAz)
+	zones, err := zone.FromSourceRegionZone(ctx, s, cli, sourceRegion, volAz)
 	if err != nil {
 		return "", "", err
 	}
 	if len(zones) != 1 {
```

Comment:
Do we need to check if `len(zones) == 0` now that we log and continue below

Reply:
I return an error if the len(zones)==0.

Reply:
gotcha.